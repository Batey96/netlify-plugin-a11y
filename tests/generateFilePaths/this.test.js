const fs = require('fs');
const path = require('path');

const BUILD_DIR = path.join(__dirname, 'publishDir');
// actual test
const pluginCore = require('../../plugin/pluginCore.js');
test('generateFilePaths works', async () => {
  const results = await pluginCore.generateFilePaths({
    fileAndDirPaths: ['/blog', '/about.html'],
    BUILD_DIR
  });
  expect(results).toMatchSnapshot();
});
