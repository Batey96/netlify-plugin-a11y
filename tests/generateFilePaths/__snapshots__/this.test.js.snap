// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateFilePaths works 1`] = `
Array [
  "/Users/swyx/Netlify/netlify-plugin-a11y/tests/generateFilePaths/publishDir/blog/post1.html",
  "/Users/swyx/Netlify/netlify-plugin-a11y/tests/generateFilePaths/publishDir/blog/post2.html",
  "/Users/swyx/Netlify/netlify-plugin-a11y/tests/generateFilePaths/publishDir/about.html",
]
`;
